docs/src/data/objects/managedcredential.yaml

```diff
@@ -0,0 +1,35 @@
+# vim: set et ts=2 sw=2 tw=80 :
+name: ManagedCredential
+prefix: managed/credentials
+description: >
+  Managed Service credential objects describe monitoring service credentials associated with a service.
+schema:
+  id:
+    type: Integer
+    value: 123
+    description: >
+      This credential's ID.
+  label:
+    type: String
+    value: CredentialLabel
+    description: >
+      A unique Label describing the credential.
+  username:
+    type: String
+    value: myusername
+    description: >
+      An optional login username.
+  last_decryption:
+    type: Object
+    description: >
+      The username and datetime of the last successful login using this credential.
```

Comment:
not sure if these are "successful" decryptions only

Reply:
I will remove the word successful.